apps/web/src/app/wrong/create/utils/uniq-by-id.ts

```
@@ -0,0 +1,5 @@
+export const uniqById = <T extends { id: string }>(list: readonly T[]) => {
+  const map = new Map<string, T>();
+  list.forEach((v) => map.set(v.id, v));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)